Update dotnet test --arch and --runtime options (#48181)
dotnet/docs: docs/core/tools/dotnet-test.md

[!INCLUDE [arch-no-a](../../../includes/cli-arch-no-a.md)]

> [!NOTE]

> > Running tests for a solution with a specific `Architecture` is not supported. If needed, this should be specified at the individual project level instead.

Reviewer: ```suggestion
> Running tests for a solution with a specific `RuntimeIdentifier` is not supported. If needed, this should be specified at the individual project level instead.
```

  Short form `-r` available starting in .NET SDK 7.

  > [!NOTE]

>   > Running tests for a solution with a global `RuntimeIdentifier` property (explicitly or via `--arch`, `--runtime`, or `--os`) is not supported. You can set `RuntimeIdentifier` on individual project level instead.

Reviewer 2: ```suggestion
  > Running tests for a solution with a global `RuntimeIdentifier` property (explicitly or via `--arch`, `--runtime`, or `--os`) is not supported. Set `RuntimeIdentifier` on an individual project level instead.
```

  Short form `-r` available starting in .NET SDK 7.

> [!NOTE]

> > Running tests for a solution with a specific `Runtime Identifier` is not supported. If needed, this should be specified at the individual project level instead.

Reviewer: This needs to be indented. I would also remove the space in "Runtime Identifier"
